src/Shared/SFA.DAS.SharedOuterApi/InnerApi/Responses/RoatpV2/ProviderDetails.cs

```diff
@@ -0,0 +1,4 @@
+﻿namespace SFA.DAS.SharedOuterApi.InnerApi.Responses.RoatpV2;
+public class ProviderDetails : Provider
```

Comment:
I dont think this adds any value, we can use Provider class in the Response object.

Reply:
ok